using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -614,32 +614,110 @@ public static void setAdvertisingIdentifier(final String advertisingIdentifier);
+
+{% endhint %}
+
+This example requires Google Play Services to be configured in your mobile application. For instructions on how to import the Google Mobile Ads SDK and how to configure your ApplicationManifest.xml file see [Google Mobile Ads SDK setup](https://developers.google.com/admob/android/quick-start#import_the_mobile_ads_sdk).
+
 ```java
-MobileCore.setAdvertisingIdentifier("advertising_identifier");
+...
+@Override
+public void onResume() {
+    super.onResume();
+    ...
+    new Thread(new Runnable() {
+        @Override
+        public void run() {
+            String advertisingIdentifier = null;
+
+            try {
+                AdvertisingIdClient.Info adInfo = AdvertisingIdClient.getAdvertisingIdInfo(getApplicationContext());
+                if (adInfo != null) {
+                    if (!adInfo.isLimitAdTrackingEnabled()) {
+                        advertisingIdentifier = adInfo.getId();
+                    } else {
+                        MobileCore.log(LoggingMode.DEBUG, "ExampleActivity", "Limit Ad Tracking is enabled by the user, cannot process the advertising identifier");
+                    }
+                }
+
+            } catch (IOException e) {
+                // Unrecoverable error connecting to Google Play services (e.g.,
+                // the old version of the service doesn't support getting AdvertisingId).
+                MobileCore.log(LoggingMode.DEBUG, "ExampleActivity", "IOException while retrieving the advertising identifier " + e.getLocalizedMessage());
+            } catch (GooglePlayServicesNotAvailableException e) {
+                // Google Play services is not available entirely.
+                MobileCore.log(LoggingMode.DEBUG, "ExampleActivity", "GooglePlayServicesNotAvailableException while retrieving the advertising identifier " + e.getLocalizedMessage());
+            } catch (GooglePlayServicesRepairableException e) {
+                // Google Play services is not installed, up-to-date, or enabled.
+                MobileCore.log(LoggingMode.DEBUG, "ExampleActivity", "GooglePlayServicesRepairableException while retrieving the advertising identifier " + e.getLocalizedMessage());
+            }
+
+            MobileCore.setAdvertisingIdentifier(advertisingIdentifier);
+        }
+    }).start();
+}
 ```
 {% endtab %}
 
 {% tab title="iOS" %}
 ### setAdvertisingIdentifier
 
+{% hint style="info" %}
+**Tip:** Retrieve the IDFA from Apple APIs only if you are using an ad service. If you retrieve
+IDFA, and are not using it properly, your app might be rejected.
+{% endhint %}
+
+{% hint style="warning" %}
+
+This is just an implementation example, for more details about IDFA and how you should handle it correctly in your mobile application, please refer to [Apple developer documentation about IDFA](https://developer.apple.com/documentation/adsupport/asidentifiermanager)
```

Comment:
Please reword so that the text reads as follows:

"This is just an implementation example. For more information about IDFA and how to handle them correctly in your mobile application, see ."